PsfLauncher/main.cpp

```diff
@@ -25,12 +63,53 @@ int launcher_main(PWSTR args, int cmdShow) noexcept try
     auto exeName = appConfig->get("executable").as_string().wide();
     auto dirPtr = appConfig->try_get("workingDirectory");
     auto dirStr = dirPtr ? dirPtr->as_string().wide() : nullptr;
+	auto monitor = PSFQueryAppMonitorConfig();
 
     // At least for now, configured launch paths are relative to the package root
     std::filesystem::path packageRoot = PSFQueryPackageRootPath();
     auto exePath = packageRoot / exeName;
     std::wstring cmdLine = L"\"" + exePath.filename().native() + L"\" " + args;
 
+if (monitor != nullptr )
+	{
+		// A monitor is an optional additional program to run, such as the PSFShimMonitor. This program is run prior to the "main application".		
+		bool asadmin = false;
+		bool wait = false;

```

Comment:
Please add a short Readme file under the psfMonitor to document the options below.

Reply:
Readme for PsfMonitor and an update for PsfLauncher (now that it has one!) included in commit 56da163